modules/configuration/src/main/java/org/apache/ignite/configuration/validation/ValidationIssue.java

```diff
@@ -0,0 +1,4 @@
+package org.apache.ignite.configuration.validation;
```

Comment:
File without a license...
You can ask me how to set up automatic license header creation so you won't have to do it manually.